src/main/webapp/app/course/shared/course-group/course-group.component.ts

```
@@ -238,50 +175,41 @@ export class CourseGroupComponent {
+        if (!courseId || !slug) {
+            return;
         }
-    };
+        this.courseManagementService.getAllUsersInCourseRole(courseId, slug).subscribe({
+            next: (res) => {
+                const users = res.body ?? [];
+                if (users.length === 0) {
+                    return;
+                }
+                const exportRows: ExportUserInformationRow[] = users.map(
+                    (user: User): ExportUserInformationRow => ({
```

ESLint

[error] Delete `⏎····················`

(prettier/prettier)